src/components/ContributingSection.tsx

```diff
@@ -5,15 +5,17 @@ import MD, { Fragment, Heading, LineBreak, Link, Text } from "jsx-md";
 /** @internal */
 interface Props {
   contributingFilePath?: string;
+  title?: string;
 }
-
+const defaultTitle = "🖋️ Contributing";
 /** Displays a contributing section based on optional contributing file path. */
 export const ContributingSection: Component<Props> = ({
   contributingFilePath = "./CONTRIBUTING.md",
+  title,
```

Comment:
Please use a default value here instead of the check inside the Heading component:
```ts
title = "🖋️ Contributing"
```

Please also do so in the other components.

Reply:
Changes done . Kindly check 0047aaf4d10c5b9718ab1001071666314dd9ce5c